components/bl-double-calendar-component/src/helpers/utils.js

```diff
@@ -13,3 +13,13 @@ export function differenceInDays(start, end) {
 
   return Math.round(diffInTime / ONE_DAY) + 1;
 }
+
+export function getShortDate(date) {
+  const dateObject = new Date(date);
+
+  const month = dateObject.toLocaleString('en-US', { month: 'short' });
+  const day = dateObject.getDate();
+  const year = dateObject.getFullYear();
+
+  return `${month} ${day} ${year}`;
```

Comment:
date.toLocaleDateString()

That will helps us to avoid wrong format depend on locale

Reply:
If do not pass the locale, it will turn out like this
<img width="332" alt="Screenshot 2023-03-20 at 19 04 16" src="https://user-images.githubusercontent.com/56361878/226414591-4d52821f-947c-48f0-b6b2-cba5828b3c5c.png">

Reply:
I think it will be better if `month` is `short` because
<img width="432" alt="Screenshot 2023-03-21 at 11 07 10" src="https://user-images.githubusercontent.com/56361878/226560815-f4011fb2-fbf8-4401-bbdb-6bf5579563ce.png">